.github/workflows/tech-debt.yml

```
@@ -32,8 +32,8 @@ jobs:
       - name: Set up Node.js
         uses: actions/setup-node@v4
         with:
-          node-version: "16"
-
+          node-version: '18'
+          
```

```suggestion
```suggestion
          node-version: '18'

```
```